packages/@react-aria/selection/stories/useSelectableList.stories.tsx

```diff
@@ -131,4 +132,54 @@ storiesOf('useSelectableList', module)
     <SelectableList isSubUlRelativelyPositioned isUlRelativelyPositioned>
       {options}
     </SelectableList>
-  ));
+  ))
+  .add(
+    'single select, allow empty, select on focus',
+    () => (
+      <List selectionMode="single">
+        <Item>Paco de Lucia</Item>
+        <Item>Vicente Amigo</Item>
+        <Item>Gerardo Nunez</Item>
+      </List>
```

Comment:
This allows you to select multiple options via moving the focus via option+arrow keys and option+Enter/Space. Same with the single select story below